lib/helpers/config.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const config = {
+  nodeEnv: process.env.NODE_ENV,
```

Comment:
Since we are accessing `process.env` multiple times, could we assign `process.env` to a variable, and then access the properties via that variable? May give us a small performance improvement as we won't need to keep accessing the `process` global in order to get to the `env` property.